motoman_mh5_support/launch/robot_interface_streaming_mh5.launch

```diff
@@ -9,10 +9,10 @@
     robot_interface_streaming_mh5.launch robot_ip:=<value> controller:=<fs100|dx100>
 -->
 <launch>
-	<arg name="robot_ip" />
+	<arg name="robot_ip" doc="IP of controller (only required if not using industrial simulator)" />
 	
   <!-- controller: Controller name (fs100 or dx100) -->
-  <arg name="controller"/>
+  <arg name="controller" doc="Name of the controller"/>
```

Comment:
This is a bit ambiguous: it's really the controller *series* or *type* that is specified by the `controller` `arg`.

Reply:
Would probably also be good to add the possible values there (ie: `dx100`, `fs100`, `dx200` and `yrc1000`).

Reply:
Modified.